src/app/index.ts

```diff
@@ -148,7 +152,7 @@ module.exports = class TheiaExtension extends Base {
                 message: "The extension's type",
                 choices: [
                     { value: ExtensionType.HelloWorld, name: 'Hello World' },
-                    { value: ExtensionType.Widget, name: 'Widget' },
+                    { value: ExtensionType.Widget, name: 'Widget (with unit tests)' },
```

Comment:
I'd drop the 'with unit tests' in the name:

```suggestion
                    { value: ExtensionType.Widget, name: 'Widget' },
```

We should update the `readme` of the generated extension to include mentions of included unit-tests and how to run them (which is already done).

Reply:
I do not have a strong opinion on this, but keeping (with unit tests) allows users to find an example for unit tests, so I prefer to keep it.

Reply:
I would leave it and remove it when other templates also have tests

Reply:
I'll leave the decision up to you, I think that is a fine proposal to only drop it when other templates also include tests.